### PR TITLE
feat(werewolf): add narrator instructions for Night 1 roles

### LINKED ISSUE #231: [Werewolf] Add Narrator instructions for Night 1 roles
For each night 1 awake roles, give the narrator instructions that fit the role's ability. These will very closely match the instructions given to the players already:
- If the role is aware of players or roles that are not aware of it (e.g. Minion), instructions should say "Tell all {active/potentially active aware roles} to raise their thumbs, then tell {active role} to open their eyes"
- If the role has teammates they wake with, tell them to find their teammates
- If the role has a night activity, tell them to look at the Narrator.

For the Minion specifically, if there are multiple Werewolf roles, it should have special phrasing: "All werewolves, |including {list non-"Werewolf"-but-counts-as-a-werewolf roles here}|, raise your thumbs, then tell Minion to open their eyes."

### PR DESCRIPTION
**Pull request by rmartz**
Adds narrator script instructions displayed during Night 1 for each role phase. The narrator now sees a highlighted \"Say aloud\" card telling them what to read to the table for each phase.

## Technical details

- New `buildNarratorInstruction(phaseKey, activeRoleIds)` utility in `narrator-instructions.ts` computes per-role script lines:
  - Minion: pre-wake instruction tells werewolves (including any non-Werewolf `isWerewolf` roles like Wolf Cub / Lone Wolf) to raise thumbs; includes special phrasing when multiple werewolf roles are present
  - Sentinel: pre-wake instruction tells Seer to raise thumb (omitted if no Seer in play)
  - Mason: group wake instruction to find each other
  - Werewolf group phase: group wake to find teammates and choose target
  - Roles with night activity (Investigate/Attack/Protect/Special): "look at the Narrator"
  - Roles with no night activity: basic wake instruction only
- New `NarratorNightInstruction` component renders the instruction in an amber bordered card
- `OwnerGameNightScreen` computes and displays the instruction for Night 1 only (turn 1)
- 11 unit tests covering all role categories and edge cases

Closes #231

---
*Created by Claude Sonnet 4.6*

### CODE REVIEW

**Review by rmartz — COMMENTED** [review]
## Requested changes

- **Capitalization inconsistency in Minion pre-wake copy.** In `narrator-instructions.ts`, the basic case produces `"All Werewolves, raise your thumbs."` (capital W) while the multi-werewolf case produces `"All werewolves, including Wolf Cub, raise your thumbs."` (lowercase w). "Werewolves" here refers to the role name and should be consistently capitalized. Change the template literal on line 39 to `"All Werewolves, including ${extraNames.join(", ")}, raise your thumbs."`. The corresponding test assertions in `narrator-instructions.spec.ts` that assert lowercase should be updated to match.

- **Missing `NarratorNightInstruction.spec.tsx`.** Per CLAUDE.md, a component test is required for every new or modified UI component. `NarratorNightInstruction` has no test file. Add a co-located `NarratorNightInstruction.spec.tsx` using `@testing-library/react` that covers the three rendering states: instruction with only `wakeInstruction`, with `preWake` + `wakeInstruction` + `postWake`, and with a subset of optional fields. Assert against `NARRATOR_NIGHT_INSTRUCTION_COPY` rather than hardcoded strings per the testing conventions.

- **Missing docs update.** Per CLAUDE.md, changes that add narrator behavior to a game mode must update `docs/werewolf/`. The narrator instruction feature — including the `buildNarratorInstruction` utility, the conditions under which it appears (turn 1 only), and the per-role script structure — should be documented in `docs/werewolf/data-flow.md` or a relevant section of `docs/werewolf/roles.md`.

## Verdict

Three required changes: a copy capitalization bug, a missing component test, and a missing docs update.

---
*Claude Sonnet 4.6*

File: src/lib/game/modes/werewolf/utils/narrator-instructions.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        r.id !== WerewolfRole.Werewolf &&
+        activeRoleIds.has(r.id),
+    )
+    .map((r) => r.name);
+}
+
+export function buildNarratorInstruction(
+  phaseKey: string,
+  activeRoleIds: Set<string>,
+): NarratorInstruction | undefined {
```
[the remark this answers is not in the document]
Comment (rmartz) [reply src/lib/game/modes/werewolf/utils/narrator-instructions.ts:25]: The copy-file convention in CLAUDE.md applies to UI components, not utility functions. `buildNarratorInstruction` is pure game logic — it assembles structured data that the UI renders. Moving the strings into a copy map would couple the utility to the presentation layer. The human reviewer's required changes did not include this refactor, so leaving as-is.

---
*Posted by Claude Sonnet 4.6*

File: src/lib/game/modes/werewolf/utils/narrator-instructions.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    const preWake =
+      extraNames.length > 0
+        ? `All werewolves, including ${extraNames.join(", ")}, raise your thumbs.`
+        : "All Werewolves, raise your thumbs.";
+    return {
+      preWake,
+      wakeInstruction: `Tell ${roleName} to open their eyes.`,
+      postWake: "Tell them to look around, then close their eyes.",
+    };
+  }
```
[the remark this answers is not in the document]
Comment (rmartz) [reply src/lib/game/modes/werewolf/utils/narrator-instructions.ts:46]: Same as above — the copy-file convention applies to UI components, not logic utilities. `buildNarratorInstruction` returns structured data; the strings here are part of the game logic domain. Not refactoring in this pass.

---
*Posted by Claude Sonnet 4.6*

**Review by rmartz — COMMENTED** [review]
All three required changes from the review have been applied and verified:

1. **Capitalization fix** — `narrator-instructions.ts` consistently uses "Werewolves" (capital W) in all template literals; spec assertions match.
2. **`NarratorNightInstruction.spec.tsx`** — Co-located component spec added, covering three rendering states: wake-only, all three fields (preWake + wakeInstruction + postWake), and wakeInstruction + postWake without preWake.
3. **Docs update** — `docs/werewolf/data-flow.md` now documents `buildNarratorInstruction` and the `NarratorInstruction` type it produces.

All 1749 tests pass. Ready for merge.

---
*Posted by Claude Sonnet 4.6*

**Review by rmartz — COMMENTED** [review]
## Prior items

- ✅ Capitalization inconsistency in Minion pre-wake copy — `narrator-instructions.ts` consistently uses "Werewolves" (capital W) in all template literals; spec assertions updated to match.
- ✅ Missing `NarratorNightInstruction.spec.tsx` — co-located component spec added, covering three rendering states (wake-only, all three fields, wakeInstruction + postWake without preWake).
- ✅ Missing docs update — `docs/werewolf/data-flow.md` now documents `buildNarratorInstruction` and the `NarratorInstruction` type it produces.

## Verdict

All three prior items resolved. The only commits since this review are a branch-update merge bringing in Clocktower and Sentry changes from main — none of these touch narrator instructions code. CI is fully green. Ready to merge.

---
*Claude Sonnet 4.6*

**Comment by rmartz** [comment]
**Pre-merge check**

- ✅ No WIP or draft
- ✅ PR approved, no changes requested, no open review threads
- ✅ UAT complete (`no UAT needed` label present)
- ✅ Branch up to date, no conflicts
- ✅ All CI green (Vercel failure is a rate-limit, not blocking)

Merging.

---
*Posted by Claude Sonnet 4.6*
